Source/Core/ComponentDatatype.js

```diff
@@ -146,7 +146,9 @@ define([
         case ComponentDatatype.DOUBLE:
             return Float64Array.BYTES_PER_ELEMENT;
         default:
+            //>>includeStart('debug', pragmas.debug);
```

Comment:
You can move this above the `default:`, since there's no reason to include it if it's empty.